course/steps/08_react-apollo/pt.md

```diff
@@ -0,0 +1,82 @@
+
+# Conectando *back-end* e *front-end*
+
+## Introdução
+
+Agora aprenderemos como recuperar dados do *backend* e exibí-los na interface. O VTEX IO utiliza [GraphQL](https://graphql.org/) como linguagem/tecnologia para transferência de dados, o que torna a programação dos nossos componentes bastante simples. Iremos modificar o nosso componente Countdown para buscar o *targetDate* do **campo `releaseDate` de um produto da VTEX**. Para realizar queries GraphQL em React, é utilizado o **Apollo Client**, uma biblioteca de gerenciamento de estado que facilita a integração de uma API GraphQL com a aplicação *front-end*.
```

Comment:
"exibi-los" instead of "exibí-los" (oxítonas terminadas em -i e -u não se acentuam na regra geral, só sendo chatinha com o português mesmo)